src/test/java/com/karankumar/bookproject/backend/entity/account/UserTest.java

```diff
@@ -61,7 +63,21 @@ void testValidUserSaved() {
                    .username("username")
                    .email("email");
     }
+    
+    @ParameterizedTest
+    @ValueSource(strings = {"123456789", "aPassWorD", "testPa$$123"})
+    void testPasswordIsInvalid(String input) {
+            // given
+            User user = userWithoutPassword().password(input)
+                                             .build();
+
+            // when
+            tryToSaveInvalidUser(user);
 
+            // then
+            assertEquals(initialNumberOfUsers, userRepository.count());
```

Comment:
Could you please change this to use an [AssertJ ](https://assertj.github.io/doc/)assertion (in line with our contributing guidelines)?

Reply:
I will check it out !
Thanks for the feedback.